app/controllers/shareconomy/api/v1/auth/registrations_controller.rb

```diff
@@ -0,0 +1,65 @@
+module Shareconomy::Api::V1::Auth
+  class RegistrationsController < DeviseTokenAuth::RegistrationsController
+    resource_description do
+      short 'Users manager'
+      api_versions 'v1'
+    end
+
+    api! 'Email registration. A verification email will be sent to the email address provided.'
+    param :email, String, required: true, desc: 'Email address'
+    param :first_name, String, required: true, desc: 'First name'
```

Comment:
The profile model has no validation for this field, but here you point it as required.